src/components/InputCode/index.tsx

```diff
@@ -37,22 +38,12 @@ const InputCode: React.FC<Props> = props => {
         }
     }, [inputIndexFocused]);
 
-    const handlePasteCode = (pasteValue: string): string => {
-        const regex = new RegExp(`\\d{${digits}}`);
-        const match = regex.exec(pasteValue);
-        if (!match) {
-            return '';
-        }
-        const [matchCode] = match;
-        return matchCode;
-    };
-
     const handleChange = async (value: string) => {
         const clipboardContent = await Clipboard.getString();
         const newValues = [...values];
         newValues[inputIndexFocused] = value;
         setValues(newValues);
-        const codeIsPasted = handlePasteCode(clipboardContent);
+        const codeIsPasted = handlePasteCode(digits, clipboardContent);
```

Comment:
we should use a more meaningful name like `code`  since codeIsPasted sounds like a boolean value and what is returned here is the code itself